hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -790,6 +795,23 @@ public boolean getReplicationManagerStatus() {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public Token<?> getContainerToken(ContainerID containerID)
+      throws IOException {
+    String remoteUser = getRemoteUserName();
+    getScm().checkAdminAccess(remoteUser);
+
+    if (!blockTokenEnabled) {
```

Comment:
should we use the container token enabled key to determine whether return a valid container token or not?

Reply:
Thanks, fixed.